x-pack/plugins/reporting/jest.integration.config.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
   preset: '@kbn/test/jest_integration',
   rootDir: '../../..',
   roots: ['<rootDir>/x-pack/plugins/reporting'],
+  collectCoverageFrom: ['<rootDir>/x-pack/plugins/reporting/**/*.{js,ts,tsx}'],
```

Comment:
This is still a line that isn't in `main`

`git checkout main x-pack/plugins/reporting/jest.integration.config.js`